DFC.App.JobProfile.CurrentOpportunities.UnitTests/DFC.App.JobProfile.CurrentOpportunities.UnitTests.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+    <CodeAnalysisRuleSet>../UnitTests.CodeAnalysis.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="TestHelpers\**" />
+    <Compile Remove="ViewTests\**" />
+    <EmbeddedResource Remove="TestHelpers\**" />
+    <EmbeddedResource Remove="ViewTests\**" />
+    <None Remove="TestHelpers\**" />
+    <None Remove="ViewTests\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="..\stylecop.json" Link="stylecop.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FakeItEasy" Version="5.2.0" />
```

Comment:
You seem to have multiple versions of the same package - check and consolidate the nuget packages

Reply:
Consolidated - think the version went up while I was putting the various parts of the solution together!